cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -78,6 +88,14 @@ func (c *CredentialsCmd) configureLogger(logger *logrus.Logger) {
 	}
 }
 
+func (c *CredentialsCmd) configureTelemetry() (t *telemetry.Telemetry, err error) {
+	t, err = telemetry.NewTelemetry(c.TelemetryAddress)
+	if err != nil && len(c.STSRegion) > 0 {
+		t.AddLabel("stsRegion", c.STSRegion)
+	}
+	return
+}
```

Comment:
We instantiate the null (as in functionality) Telemetry object pre-parsing so this is our hook to create a real one based off of various command line arguments.